examples/seismic/tti/wavesolver.py

```diff
@@ -52,7 +52,7 @@ def op_fwd(self, kernel='centered', save=False):
         """Cached operator for forward runs with buffered wavefield"""
         return ForwardOperator(self.model, save=save, geometry=self.geometry,
                                space_order=self.space_order,
-                               kernel=kernel, **self._kwargs)
+                               **self._kwargs)
```

Comment:
NOTE FR: This kernel arg causes memoization error if I keep it.
NOt sure how to solve it.

Reply:
WDYM memoization error, please put it back so can be debugged it should be there (and in adjoint as well)